documentation/asciidoc/topics/proc_configuring_pod_probes.adoc

```diff
@@ -0,0 +1,26 @@
+[id='configuring-pod-probes{context}']
```

Comment:
file needs to be linked somewhere, right?

Reply:
It would definitely help :slightly_smiling_face: